securedrop/management/run.py

```diff
@@ -151,12 +154,15 @@ def run():
 
     """
     procs = [
-        DevServerProcess('Source Interface',
-                         ['python', 'source.py'],
-                         'blue'),
-        DevServerProcess('Document Interface',
-                         ['python', 'journalist.py'],
-                         'cyan'),
+        lambda: DevServerProcess('Source Interface',
+                                 ['python', 'source.py'],
+                                 'blue'),
+        lambda: DevServerProcess('Document Interface',
+                                 ['python', 'journalist.py'],
+                                 'cyan'),
+        lambda: DevServerProcess('SASS Compiler',
+                                 ['sass', '--watch', 'sass:static/css'],
+                                 'magenta'),
```

Comment:
If one of the subprocesses fails to start or throws and exception, the monitor will leave the other processes dangling. Each call to DevServerProccess needs a try/except around it.